.github/workflows/build.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+    types: [ 'opened', 'reopened', 'edited', 'synchronize' ]
+  workflow_dispatch:
+
+jobs:
+  cancel_previous:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          workflow_id: ${{ github.event.workflow.id }}
+
+  braze-test:
+    needs: cancel_previous
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: cache gradle dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-core-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-core-
+      - name: Run Tests
+        run: ./gradlew braze:test
+#      - name: Generate coverage report
+#        run: ./gradlew braze:codeCoverageReport
+#      - name: Upload coverage to Codecov
+#        uses: codecov/codecov-action@v2
+
+#  android-test:
+#    needs: cancel_previous
+#    runs-on: ubuntu-latest
+#
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Grant execute permission for gradlew
+#        run: chmod +x gradlew
+#      - name: cache gradle dependencies
+#        uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.gradle/caches
+#            ~/.gradle/wrapper
+#          key: ${{ runner.os }}-gradle-android-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+#          restore-keys: |
+#            ${{ runner.os }}-gradle-android-
+#      - name: Run Tests
+#        run: ./gradlew android:test
+#      - name: Generate coverage report
+#        run: ./gradlew android:codeCoverageReport
+#      - name: Upload coverage to Codecov
+#        uses: codecov/codecov-action@v2
+
+#  security:
+#    needs: cancel_previous
+#    runs-on: ubuntu-latest
+#
+#    steps:
```

Comment:
you can remove this snyk is added without as having to create an actionfrom the platform is self